pnpm-lock.yaml

```diff
@@ -18256,6 +18585,17 @@ packages:
       fsevents: 2.3.3
     dev: false
 
+  /tsx@4.3.0:
```

Comment:
Might be worth a dedupe, not sure these duplicates are necessary or not

Reply:
The one above is `tsx@3`